blaze/expr/tests/test_expr.py

```diff
@@ -136,3 +136,15 @@ def test_hash_to_different_values():
     from blaze.expr.expressions import _attr_cache
     assert (expr, '_and') in _attr_cache
     assert (expr2, '_and') in _attr_cache
+
+
+def test_coerce():
+    s = symbol('s', var * float32)
+    assert str(s.coerce('int64')) == "s.coerce(to='int64')"
+
+
+@pytest.mark.xfail(raises=AttributeError, reason='Should this be valid?')
```

Comment:
Is the question here if we should be able to cast down from float64 to float32? I think the idea of casting records like this is pretty cool.